site/en/guide/keras.md

```diff
@@ -132,4 +124,4 @@ To learn more about Keras, see the following topics at
 * [Introduction to Keras for Engineers](https://keras.io/getting_started/intro_to_keras_for_engineers/)
 * [Introduction to Keras for Researchers](https://keras.io/getting_started/intro_to_keras_for_researchers/)
```

Comment:
This link was broken and I couldn't find the proper page to use. Any suggestions?

Reply:
Can we change it to an archive link? Or fully remove should also work.